### PR TITLE
Add errors for PolicyBuilder

### PR DESCRIPTION
**Pull request by oanatmaria**

### CODE REVIEW

File: cerr/errors.go [hunk 1]
```diff
@@ -119,6 +119,10 @@ var (
 	ErrQueryExecutionFailed = newErr("E10048", codes.FailedPrecondition, http.StatusBadRequest, "query failed")
 	// Returned when the account has not setup a personal tenant yet
 	ErrPersonalTenantRequired = newErr("E10049", codes.FailedPrecondition, http.StatusBadRequest, "personal tenant required")
+	// Returned if a policy builder id is not found in the database
+	ErrPolicyBuilderNotFound = newErr("E10050", codes.NotFound, http.StatusNotFound, "policy builder not found")
+	// Returned if a policy builder id is invalid
+	ErrInvalidPolicyBuilderID = newErr("E10051", codes.InvalidArgument, http.StatusBadRequest, "invalid policy builder id")
```
Comment (gimmyxd) [inline cerr/errors.go:125]: Isn't the policy builder id something that we generate? when could this be invalid?

Comment (oanatmaria) [reply cerr/errors.go:125]: When it's provided by the user on delete request

Comment (gimmyxd) [reply cerr/errors.go:125]: Why not return `ErrPolicyBuilderNotFound` in that case?

Comment (oanatmaria) [reply cerr/errors.go:125]: Do be able to differentiate between the case when a PolicyBuilder ID is not in the database & a PolicyBuilder ID is incorrect

**Review by viovanov — APPROVED** [review]

**Pull request merged**